tests/middleware_exceptions/tests.py

```diff
@@ -149,7 +149,7 @@ def test_log(self):
         with self.assertLogs('django.request', 'DEBUG') as cm:
             self.client.get('/middleware_exceptions/view/')
         self.assertEqual(
-            cm.records[0].getMessage(),
+            cm.records[1].getMessage(),
```

Comment:
Why did this index change? Just curious.

Reply:
The `Synchronous middleware adapted` log message is in `cm.records[0]`.

Reply:
Aha! Right.